libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -88,6 +88,8 @@ import com.salesforce.androidsdk.auth.OAuth2.revokeRefreshToken
 import com.salesforce.androidsdk.auth.idp.SPConfig
 import com.salesforce.androidsdk.auth.idp.interfaces.IDPManager
 import com.salesforce.androidsdk.auth.idp.interfaces.SPManager
+import com.salesforce.androidsdk.auth.nativeLoginInterface.NativeLoginManager as NativeLoginManagerInterface
```

Comment:
This little trick makes the code super readable.

Reply:
You are right on the package name:
```
Names of packages are always lowercase and do not use underscores (org.example.project). Using multi-word names is generally discouraged, but if you do need to use multiple words, you can either just concatenate them together or use the camel case (org.example.myProject).
```
`.native.login.interfaces` is a great suggestion, I see you did that with local notifications.  I'll follow that pattern here too.

Personally I hate class names with `Default` (when there is only one impl) or `Internal` in them because they are just generic junk added on to make the name unique. Might as well by `NativeLoginManager1` for all the value they add lol.  

The reason I like this naming convention is because it is the clearest naming for us **and** the consumer simultaneously.  `auth.native.login.interface.NativeLoginManager` is an interface but to the consumer it is also the implementation.  When we return an instance of that interface to them they get a concrete implementation.  But internally it's just an interface to us, so being able to refer to it as `NativeLoginManagerInterface` internally and have `auth.NativeLoginManager` keep it's name is the best of both worlds imo.